src/main/java/chatbot/task/Deadline.java

```diff
@@ -0,0 +1,40 @@
+package chatbot.task;
+
+import chatbot.datetime.Timestamp;
+
+
+/**
+ * Represents a Task with a Timestamp by which it must be completed.
+ */
+public class Deadline extends Task {
+
+    private final Timestamp by;
+
+    /**
+     * Instantiates a new Deadline.
+     *
+     * @param title The title of the deadline.
+     * @param by    The timestamp of the deadline.
```

Comment:
The spaces between the parameter description and the parameter name seems to be inconsistent, based on style guide I think description should be a single space away from the parameter name - https://se-education.org/guides/conventions/java/intermediate.html#comments